modules/ensemble/lib/widget/carousel.dart

```diff
@@ -100,6 +101,8 @@ class Carousel extends StatefulWidget
       'direction': (value) =>
           _controller.direction = Utils.optionalString(value),
       'cacheKey': (value) => _controller.cacheKey = Utils.optionalString(value),
+      'scrollType': (value) =>
+          _controller.scrollType = CarouselScrollPhysics.values.from(value),
```

Comment:
check (CarouselScrollPhysics.values.contains(value))  before setting it. We don't want to throw an error. If the value is not valid, set it to null.